docs/misc/creating-enrichments.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Creating Enrichments
+nav_order: 1
+parent: Miscellaneous information
+---
+
+# Adding an enrichment to a collection
+
+## create a collection and add documents to it
+1. Create a new Project, and select the project type to be *document retrieval*
+2. Next, select *Upload data*
+3. Name the collection _Collection1_ and create the collection
+4. Next upload the sample document in this repo at _examples/discovery-search-app/sample-files/watson-in-the-game.pdf_
```

Comment:
Nitpicky detail, but you can probably get rid of the "Next"s in these steps, since we're already in a numbered list